src/events/winston.js

```diff
@@ -0,0 +1,56 @@
+const tracer = require('../tracer.js');
+const moduleUtils = require('./module_utils.js');
+
+/**
+ * Wrap bunyan logs
+ * @param {Function} wrappedFunction The function to wrap from bunyan
+ * @returns {function} emit wrapper function
+ */
+function writeWrapper(wrappedFunction) {
+    return function internalWriteWrapper(chunk, encoding, callback) {
+        if (!tracer.isLoggingTracingEnabled()) {
+            return wrappedFunction.apply(this, [chunk, encoding, callback]);
+        }
+        const traceId = tracer.getTraceId();
+        if (!traceId) {
+            return wrappedFunction.apply(this, [chunk, encoding, callback]);
+        }
+
+        if (!chunk || typeof chunk !== 'object') {
+            return wrappedFunction.apply(this, [chunk, encoding, callback]);
+        }
+
+        const newChunk = {
+            epsagon: {
+                trace_id: traceId,
+            },
+        };
+        /* eslint-disable guard-for-in, no-restricted-syntax */
+        for (const key in chunk) {
+            newChunk[key] = chunk[key];
+        }
```

Comment:
for this case, you can do
const newChunk = {
           epsagon: {
                trace_id: traceId,
            },
            ... chunk
}

Reply:
or maybe because we have some node 6 issues,
maybe you can do it with Object.assign() approach.

Reply:
can't becsue I want to copy non-enumerable symbols as well